app/src/main/java/com/github/pockethub/ui/MainActivity.java

```diff
@@ -86,7 +87,11 @@
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        Bugsnag.init(this);
+
+        if (!BuildConfig.DEBUG) {
```

Comment:
I'd prefer using this one: https://github.com/bugsnag/bugsnag-android#setnotifyreleasestages